src/main/java/org/zalando/nakadi/throttling/SampledTotal.java

```diff
@@ -0,0 +1,97 @@
+package org.zalando.nakadi.throttling;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SampledTotal {
+
+    private int current = 0;
+
+    private List<Sample> samples;
+    private final long windowMs;
+    private final int samplesCount;
+    private final long expireAge;
+
+    public SampledTotal(final long windowMs, final int samplesAmount) {
```

Comment:
Keep last 'samplesAnount' samples. Period of sample aggreagtion equals 'windowMs' millisecons